libirods_rule_engine_plugin-indexing.cpp

```diff
@@ -237,9 +239,11 @@ namespace {
                 const std::string operation{avu_inp->arg0};
                 const std::string type{avu_inp->arg1};
                 const std::string collection_name{avu_inp->arg2};
-                const std::string attribute{avu_inp->arg3};
-                const std::string value{avu_inp->arg4};
-                const std::string units{avu_inp->arg5};
+
+                const std::string attribute{ NULL_PTR_GUARD(avu_inp->arg3) };
+                const std::string value{     NULL_PTR_GUARD(avu_inp->arg4) };
```

Comment:
see my previous comment.

Reply:
@korydraughn I see your point about the units only thing. Do we have an error code for invalid empty AVU field?

Reply:
`SYS_INVALID_INPUT_PARAM` ?

Reply:
I can live with that.